newspaper/images.py

```diff
@@ -167,6 +171,52 @@ def fetch_image_dimension(url, useragent, referer=None, retries=1):
     return fetch_url(url, useragent, referer, retries, dimension=True)
 
 
+def valid_image_url(url):
+
+    url = clean_url(url)
+    if not url.startswith(('http://', 'https://')):
```

Comment:
But what if the input URL can be a relative URL?

Reply:
good call